css-transitions/styles.css

```diff
@@ -0,0 +1,54 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: sans-serif;
+  background-color: rgb(245, 245, 245);
+}
+
+section {
+  max-width: 992px;
+  margin: 2rem auto;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.custom-button {
+  padding: 0.5rem 1.5rem;
+  border: none;
+  border-radius: 0.25rem;
+  font-size: 1rem;
+  color: white;
+  background-color: blue;
+  transition: background-color 0.5s ease-in;
+}
+
+.custom-button:hover {
+  background-color: red;
+}
+
+.card-wrapper {
+  text-align: center;
+}
+
+.card {
+  max-width: 18rem;
+  padding: 0.5rem;
+  margin-bottom: 2rem;
+  border: 1px solid rgb(228, 228, 228);
+  border-radius: 0.25rem;
+  background-color: white;
+  cursor: pointer;
+  transform: scale(1);
+}
+
+.card:hover {
+  transition-duration: 0.3s;
```

Comment:
`transition` should be on the `.card` class, not on the pseudoclass. this ensures that the transition effect is played both when you start hovering and when you stop hovering over the element.

Reply:
@kurtyazdizadeh Fixes made and committed 😄